OpenRA.Mods.Common/Traits/AutoTarget.cs

```diff
@@ -325,6 +329,10 @@ Target ChooseTarget(Actor self, AttackBase ab, Stance attackStances, WDist scanR
 					if (ati.Priority < chosenTargetPriority)
 						return false;
 
+					// Incompatible stances
+					if (!self.Owner.Stances[owner].HasStance(ati.ValidStances))
```

Comment:
This is the wrong way around.  You want `ati.ValidStances.HasStance(self.Owner.Stances[owner])`